config/getdate.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+# Copyright (c) 2017      Research Organization for Information Science
+#                         and Technology (RIST). All rights reserved.
+
+date --date="@${SOURCE_DATE_EPOCH:-$(date +%s)}"
```

Comment:
It might be good to put a comment here in this file explaining why it exists.  Otherwise, Future People will wonder why there's a 1-line script to run the `date` command.

It might actually be good to put a 1-line comment in most/all places we do `$SOURCE_DATE_EPOCH`, referring to reproducible builds, just so that no on accidentally removes the `$SOURCE_DATE_EPOCH` because they don't understand what it's for.